cmd/vic-machine/validate/validator.go

```diff
@@ -58,8 +58,18 @@ func (v *Validator) Validate(input *data.Data) (*metadata.VirtualContainerHostCo
 	vchConfig.ApplianceSize.Memory.Limit = int64(input.MemoryMB)
 	vchConfig.Name = input.DisplayName
 
+	var errs []string
 	if err := v.ParseComputeResourcePath(input.ComputeResourcePath); err != nil {
-		return nil, err
+		errs = append(errs, err.Error())
+		connected, err := v.getConnection(input) // Connect so we can suggest valid values
+		if err != nil {
+			errs = append(errs, err.Error())
+		}
+		if connected == true {
+			log.Info("Suggesting valid values for compute-resource")
+			v.suggestAllResourcePools()
```

Comment:
With the above changes, you can move the suggest methods to getResources method.